cnext_server/server/python/libs/message.py

```diff
@@ -15,8 +15,11 @@ class WebappEndpoint(str, Enum):
     ExperimentManager = 'ExperimentManager'
     ExecutorManager = 'ExecutorManager'
     ModelManager = 'ModelManager'
+    EnvManager = 'EnvManager'
```

Comment:
let use full name `EnvironmentManager`. Code is easier to read and the name will be auto-filled anyway.